cognite/client/data_classes/assets.py

```diff
@@ -203,7 +203,7 @@ def dump(self, camel_case: bool = False) -> Dict[str, Any]:
         return result
 
     def to_pandas(
-        self, expand: Sequence[str] = ("metadata", "aggregates"), ignore: List[str] = None, camel_case: bool = True
```

Comment:
Changing defaults here should be noted in the changelog as a breaking change.

Reply:
On it ⚡

Reply:
Done, https://github.com/cognitedata/cognite-sdk-python/pull/1047/files#diff-1d2462c0f2a2dfee857bfde4966c1f353c6555def8165897cf15ff064fdea090

Reply:
Done in #1049